src/lib/Containers/Inquiry.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { MetadataText, SmallHeadline } from "../Components/Inbox/Typography"
+
+import {
+  FlatList,
+  ImageURISource,
+  KeyboardAvoidingView,
+  NativeModules,
+  StatusBar,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  ViewProperties,
+} from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../data/colors"
+import fonts from "../../data/fonts"
+
+import BottomAlignedButton from "../Components/Consignments/Components/BottomAlignedButton"
+
+import ArtworkPreview from "../Components/Inbox/Conversations/Previews/ArtworkPreview"
+import ARSwitchBoard from "../NativeModules/SwitchBoard"
+import { gravityURL } from "../relay/config"
+import { NetworkError } from "../system/errors"
+
+const Container = styled.View`
+  flex: 1
+  flex-direction: column
+`
+const Header = styled.View`
+  alignSelf: stretch
+  margin-top: 10
+  flex-direction: column
+  margin-bottom: 30
+`
+// This is really rubbish, but I basically have to create an equally sized element
+// on the top right, to get the title in the middle
```

Comment:
Alas, there's nothing official stating it, but I couldn't get it to work any other way with css / flex. I spent longer on this than I care to admit and then decided to do the simplest / silliest thing which was this...